src/api_authentication_app_skeleton/src/actions/api/sign_in/create.cr

```diff
@@ -0,0 +1,14 @@
+class Api::SignIn::Create < ApiAction
```

Comment:
To make it consistent can this be pluralized? Even though it's only used once that is the typical style.